nixos/lib/make-options-doc/default.nix

```
@@ -88,8 +88,49 @@ let
   # Convert the list of options into an XML file.
   optionsXML = builtins.toFile "options.xml" (builtins.toXML optionsList);
 
+  optionsNix = builtins.listToAttrs (map (o: { name = o.name; value = removeAttrs o ["name" "visible" "internal"]; }) optionsList);
+
+  # TODO: declarations: link to github
+  singleAsciiDoc = name: value: ''
+    == ${name}
+
+    ${value.description}
+
+    [discrete]
+    === details
+
+    Type:: ${value.type}
+    ${ if pkgs.lib.hasAttr "default" value
```

```suggestion
    ${ if lib.hasAttr "default" value
```
and in other places: "pkgs.lib" => "lib"